src/components/TransformDelimiterSeparatedValues.tsx

```diff
@@ -41,12 +43,15 @@ export const TransformDelimiterSeparatedValues = ({
 	);
 	const exportToDsv = useCallback(() => generatedDSV, [generatedDSV]);
 
+	useEffect(()=>{
+		setShowButtons(()=>processed.length>1);
```

Comment:
```suggestion
		setShowButtons(processed.length > 1);
```

I don't believe you need this to be a function call, it'll evaluate to a boolean by itself. It only makes sense to use a function in a state setter if the previous state is needed